spec/features/display_modes/single_page_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+module Quby
+  describe "quby/answers/single_page/panel" do
+    let(:questionnaire) { inject_questionnaire("panels", <<-END) }
+      title "Questionnaire with a bunch of panels"
+
+      panel do
+        text "First panel"
+      end
+
+      panel do
+        text "Second panel"
+      end
+
+      panel do
+        text "Third panel"
+      end
+    END
+
+    it 'renders the questionnaire on one page' do
+      visit_new_answer_for(questionnaire, "single_page")
+      @questionnaire = questionnaire
+      page.should_not have_selector(".step1")
```

Comment:
Heb je deze test rood gehad?
Er zou namelijk nooit .step1 in moeten staan, hooguit .step-1. 

Maar in een feature spec, liever iets betekenisvollers testen, zoals dat de text van alle 3 panels zichtbaar is.
Dat als we nooit subquestions willen verbergen, dingen niet verborgen zijn. 
Dat je validaties idd mag skippen.
Dat gefaalde ajax requests idd weer een correcte toestand opleveren.